lib/forklift/installer.rb

```diff
@@ -96,7 +96,7 @@ def run_installer(command)
 
     def syscall(command)
       system(command)
-      $CHILD_STATUS.zero?
+      $CHILD_STATUS.to_i.zero?
```

Comment:
Do you want this to return true when $CHILD_STATUS is nil? I ask because previously (`$CHILD_STATUS == 0`) it would return false for nil.